tests/visualization_tests/test_parallel_coordinate.py

```diff
@@ -348,13 +348,13 @@ def test_color_map(direction: str) -> None:
         assert not line["reversescale"]
 
     # When `target` is not `None`, `reversescale` is always `True`.
-    line = plot_parallel_coordinate(study, target=lambda t: t.number).data[0]["line"]
+    line = plot_parallel_coordinate(study, target=lambda t: float(t.number)).data[0]["line"]
```

Comment:
IMO, it is a bit too strict to reject `int` values although the type of the `target` requires `float`. How about accepting `int` as the original code intended?

c.f., `study.tell` accepts the float-castable values, and I'd like to make them consistent, personally.
https://github.com/optuna/optuna/blob/fc5e262042a60bbcc888541ce7fa5e08769ac478/optuna/study/_tell.py#L65-L71

Reply:
Agreed. Thank you for sharing a similar code. I've updated the using the same logic in `optuna/optuna/study/_tell.py`.

Reply:
Thanks!